Gems/Camera/Code/Source/ViewportCameraSelectorWindow_Internals.h

```diff
@@ -25,38 +23,26 @@ namespace Camera
         // Each item in the list holds the camera's entityId and name
         struct CameraListItem
             : public AZ::EntityBus::Handler
-            , public Maestro::SequenceComponentNotificationBus::Handler
         {
         public:
             CameraListItem(const AZ::EntityId& cameraId);
-            // Used for a virtual camera that is really whatever camera is being
-            // used for a Track View Sequence.
-            CameraListItem(const char* cameraName, const AZ::EntityId& sequenceId);
             ~CameraListItem();
 
             void OnEntityNameChanged(const AZStd::string& name) override { m_cameraName = name; }
 
-            //////////////////////////////////////////////////////////////////////////
-            /// Maestro::SequenceComponentNotificationBus::Handler
-            void OnCameraChanged(const AZ::EntityId& oldCameraEntityId, const AZ::EntityId& newCameraEntityId) override;
             bool operator<(const CameraListItem& rhs);
 
             AZ::EntityId m_cameraId;
             AZStd::string m_cameraName;
-            AZ::EntityId m_sequenceId;
         };
 
         // holds a list of camera items
         struct CameraListModel
             : public QAbstractListModel
             , public CameraNotificationBus::Handler
-            , public Maestro::EditorSequenceNotificationBus::Handler
         {
         public:
-
-            static const char* m_sequenceCameraName;
-
-            CameraListModel(QObject* myParent);
+            CameraListModel(QWidget* myParent);
```

Comment:
nit: Could we please add `explicit` here to prevent any weird implicit conversions